wp-content/themes/core/pcss/components/button/_solid.pcss

```diff
@@ -40,6 +40,10 @@ a[data-mce-selected].c-btn {
 		border: 1px solid var(--color-submarine);
 		color: var(--color-nevada);
 	}
+
```

Comment:
What is this `.c-btn__text` for? Are we missing a particular button style for light vs. dark?

Reply:
Yeah, the light/dark kitchen sink variations don't account for this child element, so I had to specify it here.

Reply:
Ahh, totally forgot we have a wrapper around label for that component 😂 Good catch!